### PR TITLE
Update latest Transifex translations

### PR DESCRIPTION
**Pull request by danesfeder**

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -1,17 +1,44 @@
 <resources>
+    <string name="app_name">SDK de mapas y navegación para Android.</string>
```
Comment (Guardiola31337) [inline app/src/main/res/values-es/strings.xml]: Are we OK adding `.` at the end of the string? The root `strings.xml` doesn't include it. The same thing happens with `description_navigation_route_ui`, `title_navigation_launcher`, `description_end_navigation`, `title_dual_navigation_map`, `description_dual_navigation_map`, `title_waypoint_navigation`, `description_waypoint_navigation`, `title_embedded_navigation`, `description_embedded_navigation`, `title_fragment_navigation`, `description_fragment_navigation`, `settings`, `simulate_route`, `route_profile`, `error_route_not_available`, `error_select_longer_route` and `explanation_long_press_waypoint`.

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -1,17 +1,44 @@
...
     <string name="description_navigation_launcher">Entra en la experiencia de la interfaz</string>
 
     <string name="title_end_navigation">Finaliza navegación</string>
+    <string name="description_end_navigation">Muestra como finalizar la navegación utilizando la vista de navegación.</string>
+
+    <string name="title_dual_navigation_map">Doble mapa de navegación.</string>
+    <string name="description_dual_navigation_map">Muestra como añadir vista de mapas y navegación en la misma disposición.</string>
+
+    <string name="title_waypoint_navigation">Punto de referencia de navegación.</string>
+    <string name="description_waypoint_navigation">Navegación con puntos de referencia entre destinos. </string>
```
Comment (Guardiola31337) [inline app/src/main/res/values-es/strings.xml]: Are we OK adding an empty space `" "` at the end of the string? The root `strings.xml` doesn't include it. The same thing happens with `title_embedded_navigation` and `error_valid_route_not_found`.

**Comment by Guardiola31337** [comment]
@danesfeder Could you pull down the latest translations from Transifex? I've just manually updated in Transifex the wrong ones commented above and should be 👌 now.

**Comment by danesfeder** [comment]
@Guardiola31337 this is updated, thanks for fixing those

**Pull request merged**